libraries/abi_generator/CMakeLists.txt

```diff
@@ -62,13 +62,3 @@ if (USE_PCH)
   set_target_properties(abi_generator PROPERTIES COTIRE_ADD_UNITY_BUILD FALSE)
   cotire(eos_utilities)
 endif(USE_PCH)
-
-install( TARGETS
-   abi_generator
-   RUNTIME DESTINATION bin
```

Comment:
Granted the ABI generator may not be operational right now, but I thought the point of it was to be a utility not unlike `eosiocpp` that could generate abi file from properly marked up contract code for the end-users.

Reply:
This is the lib, not the abi_gen exec.

Reply:
This is the abi_generator static library; eosio-abigen is the executable that users would use (that links to this static library).

Reply:
touche